SignalBackgroundMerger.cpp

```diff
@@ -625,6 +662,9 @@ class SignalBackgroundMerger {
       int status = particle->status();
       if (status == 1 ) finalParticleCount++;
       int pid = particle->pid();
+      if ((status == 1) || (status == 2)){
```

Comment:
Would make sense to do it unconditionally. There is no benefit to force analyzers of full event records to adjust only baseStatus + 1 and baseStatus + 2 codes.

Reply:
How is this relevant? The codes will be the same `--physics.alternativeStableStatuses=[2001, 4001, 6001] --physics.alternativeDecayStatuses=[2002, 4002, 6002]`, the other particles will remain invisible to ddsim regardless of whether this `if` is here.